tests/test_pipeline/test_autoregressive_pipeline.py

```diff
@@ -54,6 +54,7 @@ def test_forecast_columns(example_reg_tsds):
     )
 
 
+@pytest.mark.xfail(reason="TSDataset 2.0")
```

Comment:
In fact it was easier to place all these files to `collect_ignore`(don't fix it, this is just a comment)